### PR TITLE
Move styles to css classes

### PR DESCRIPTION
**Pull request by Zen4All**
This PR is ment to move a lot of the "style" attributes in to CSS classes. For now I limited this to:
- text color
- font weight
- vertical alignment

Changes are only made on the admin files.
The new classes have been added to the stylesheet

### CODE REVIEW

File: admin/includes/css/stylesheet.css [hunk 1]
```diff
@@ -835,3 +835,49 @@ input.faint {
...
+.txt-pink {
+  color: #ffbaba;
+}
+.txt-transparent {
+  color: transparent;
+}
+.bold {
+  font-weight: bold;
+}
+.valign-middle {
```
Comment (lat9) [inline admin/includes/css/stylesheet.css:872]: Could these various `.valign-` use, instead, the [Bootstrap 4](https://getbootstrap.com/docs/4.0/utilities/vertical-align/) names?  That will make it easier to migrate the admin from BS-3.

File: admin/includes/css/stylesheet.css [hunk 2]
```diff
@@ -835,3 +835,49 @@ input.faint {
...
+.txt-white {
+  color:#fff;
+}
+.txt-pink {
+  color: #ffbaba;
+}
+.txt-transparent {
+  color: transparent;
+}
+.bold {
```
Comment (lat9) [inline admin/includes/css/stylesheet.css:869]: The Bootstrap 4 class for bold-text is `.font-weight-bold`; could we use that instead?

**Review by lat9 — CHANGES_REQUESTED** [review]
In cases where it's possible (as a pseudo-future-proofing step), I'll suggest that if there's a Bootstrap-4 class associated, that it be used to make a future admin transition to BS4 a tad less onerous.

File: admin/includes/css/stylesheet.css (hunk 1 above)
Comment (Zen4All) [reply admin/includes/css/stylesheet.css:872]: Good plan, I had been looking for standard BS3 classes, but did not think of BS4. Will make the adjustments shortly

**Comment by drbyte** [comment]
It's worth also looking at BS5 syntax as well in case there are relevant patterns used there.  

Example: (contrived, because I've NOT checked BS5 for color-related syntax) if `txt-orange` as proposed here actually has a standard in BS5 as `text-orange` then let's use the BS5 style, even if there isn't one for it in BS4.  Again, I'm only using txt-orange for explanation purposes.

And **I don't mean to create additional work**, but if there's no standard for one of these in BS4 or BS5 then a quick check in Tailwind would be sensible, just to try to keep in sync with strong contemporary approaches, which makes future work less onerous and makes it faster for people to get up-to-speed when using ZC.

**Comment by drbyte** [comment]
@Zen4All I like what you're doing here. Thanks!

**Comment by Zen4All** [comment]
I have not had any look at BS5 yet, but will do in the coming days. But it may well be worth it. Maybe we can also skip BS4.

**Comment by proseLA** [comment]
i am a BIG fan of what you are doing here as well.  THANK YOU!

:+1:

**Pull request merged**